backend/src/main/java/org/globe42/web/security/JwtHelper.java

```diff
@@ -0,0 +1,48 @@
+package org.globe42.web.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Component;
+
+/**
+ * TODO include class javadoc here
+ * @author JB Nizet
+ */
+@Component
+public class JwtHelper {
+
+    public static final String DEFAULT_SECRET_KEY = "ng2ponyracerByNinjaSqu4d!";
```

Comment:
probably shouldn't be that token. And this is a public repository, so I an not sure it's a good idea to push the key...